cli/Cargo.toml

```diff
@@ -80,6 +80,7 @@ rpassword = { workspace = true }
 sapling-renderdag = { workspace = true }
 scm-record = { workspace = true }
 serde = { workspace = true }
+serde_json.workspace = true
```

Comment:
nit: can you copy the current style `{ workspace = true }` for consistency? (or add a separate patch that updates everything.)

Reply:
Oops, I think I added it with `cargo add` and didn't pay attention.